### ISSUE TITLE
Fix ollama client TODOs

### ISSUE THREAD

**Issue by peterychang** [microsoft/autogen#5597]
### What feature would you like to be added?

Several TODOs in ollama client

### Why is this needed?

necessary to fully support the client

**Comment by codelion** [comment]
You can try using optillm - https://github.com/codelion/optillm it already supports logits for local inference and implements several test-time-compute techniques that can benefit autogen.

**Comment by victordibia** [comment]
@peterychang , are these fixed in #5595 ?

**Comment by peterychang** [comment]
> [@peterychang](https://github.com/peterychang) , are these fixed in [#5595](https://github.com/microsoft/autogen/issues/5595) ?

No, these are most of the things that weren't properly implemented or tested in the initial client release
